.github/workflows/io-test.yml

```diff
@@ -74,6 +74,7 @@ jobs:
 
       - name: Install neo
         run: |
+          pip install .[test]
```

Comment:
shouldn't this go after the next line? Or just add `[test]` at the end of the next line?

Reply:
I did not wanted to upgrade the [test] dependencies.
It can be put after.